crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -14,23 +14,22 @@
 # limitations under the License.
 #
 
-class Api::StoragesController < ApiController
+class Api::CephController < ApiController
```

Comment:
added to cleanup card https://trello.com/c/8p1DvJQ2/1931-cleanup-upgrade-related-api as we have to change it in multiple places (in other files/repos as well)